app/models/work.rb

```diff
@@ -0,0 +1,33 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+  
+  validates :title, presence: true
+  validates :category, presence: true
+  validates :creator, presence: true
+  validates :description, presence: true
+  validates :publication_year, presence: true, numericality: { only_integer: true } 
+
+  
+  def self.media_spotlight
+    @spotlight = Work.all.sort_by { |work| work.votes.count }
+    
+    return @spotlight.last
+  end
+  
+  def self.movies
```

Comment:
Consider whether you could write a single method that takes category as an argument to DRY up your code.